test/net/java/sip/communicator/impl/protocol/jabber/extensions/colibri/ColibriIQProviderTest.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Jitsi, the OpenSource Java VoIP and Instant Messaging client.
+ *
+ * Copyright @ 2015 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.java.sip.communicator.impl.protocol.jabber.extensions.colibri;
+
+import junit.framework.TestCase;
+import net.java.sip.communicator.impl.protocol.jabber.SmackV3InteroperabilityLayer;
+import net.java.sip.communicator.service.protocol.jabber.AbstractSmackInteroperabilityLayer;
+import org.jivesoftware.smack.packet.IQ;
+import org.xmlpull.v1.XmlPullParser;
+import org.xmlpull.v1.XmlPullParserException;
+import org.xmlpull.v1.XmlPullParserFactory;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ColibriIQProviderTest extends TestCase
+{
+    String testXml = "\n" +
```

Comment:
Since this is simply for testing, how about this:
1. parse the XML from the file using a full-blown parser
2. the write it out again without whitespace/newlines
3. let XmlPullParser read from the output of 2.